float8_experimental/fsdp_utils.py

```diff
@@ -0,0 +1,58 @@
+import math
+import warnings
+from typing import List
+
+import torch
+import torch.nn as nn
+from float8_experimental.float8_dynamic_utils import WeightWithDynamicFloat8CastTensor
+from float8_experimental.float8_linear import Float8Linear
+from float8_experimental.float8_linear_utils import linear_requires_sync
+from float8_experimental.float8_utils import EPS
+
+
+def precompute_float8_amax_for_fsdp(module: nn.Module) -> None:
+    """
```

Comment:
improve docstring with example API usage

Reply:
nice! can we add this to the README?

Reply:
just added API usage to README

Reply:
nit: maybe we can make sure `dynamic` is in the name, since this is specific to dynamic scaling?

Reply:
renaming to `precompute_float8_dynamic_scale_for_fsdp`